src/omnisharp/extension.ts

```diff
@@ -74,6 +76,8 @@ export function activate(context: vscode.ExtensionContext, reporter: TelemetryRe
         localDisposables.push(vscode.languages.registerCodeActionsProvider(documentSelector, codeActionProvider));
         localDisposables.push(reportDiagnostics(server, reporter, advisor));
         localDisposables.push(forwardChanges(server));
+
+        taskProvider.activate(context, server);
```

Comment:
taskProvider is disposable, should it get added to `localDisposables`?

Reply:
done